pkgs/development/tools/build-managers/xmake/default.nix

```diff
@@ -1,48 +1,29 @@
-{ lib
-, stdenv
-, fetchurl
-, pkg-config
-, lua
-, readline
-, ncurses
-, lz4
-, tbox
-, xmake-core-sv
+{
+  lib,
+  stdenv,
+  fetchurl,
+  CoreServices,
+  ...
 }:
-
 stdenv.mkDerivation rec {
   pname = "xmake";
-  version = "2.9.4";
-
+  version = "v2.9.5";
   src = fetchurl {
-    url = "https://github.com/xmake-io/xmake/releases/download/v${version}/xmake-v${version}.tar.gz";
-    hash = "sha256-deLd4r0qSKMymJuAGuZQd8RS1JH+xRep2yeoHIcTzcU=";
+    url = "https://github.com/xmake-io/xmake/releases/download/${version}/xmake-${version}.tar.gz";
+    hash = "sha256-A/61eH4i+rjdQEGew9hKvTWrzZ+KGyTEiMfrVx1nJMg=";
   };
 
-  nativeBuildInputs = [
-    pkg-config
-  ];
-
-  buildInputs = [
-    lua
-    lua.pkgs.cjson
-    readline
-    ncurses
-    lz4
-    tbox
-    xmake-core-sv
-  ];
+  patches = [ ./0001-Fix-platform-compatibility-for-_os_find-and-_os_date.patch ];
```

Comment:
This line can be removed after upgrading to 2.9.6. See https://github.com/xmake-io/xmake/pull/5655

Reply:
Use `fetchpatch`  
https://github.com/NixOS/nixpkgs/blob/master/doc/build-helpers/fetchers.chapter.md